app/controllers/pages_controller.rb

```diff
@@ -3,6 +3,11 @@ class PagesController < ApplicationController
 
   def home
     @jobs_search_form = VacancyAlgoliaSearchForm.new
+    @vacancy_facets = VacancyFacets.new
+    @job_roles_facets = @vacancy_facets.get(:job_roles)
+    @subjects_facets = @vacancy_facets.get(:subjects).first(10)
```

Comment:
`.first(10)` happens already in the service. same for the following lines

Reply:
I thought I had removed this. My mistake!